LoopKitUI/Views/LabeledDateView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  LabeledDateView.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-02-20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct LabeledDateView: View {
+    var label: String
+    var date: Date?
+    var dateStyle: DateFormatter.Style
+    var timeStyle: DateFormatter.Style
+    
+    private var dateFormatter: DateFormatter {
+        let formatter = DateFormatter()
```

Comment:
[nit] This looks innocuous, but allocating a dateformatter is a pretty heavy operation: date formatters call up construction of calendar, locale, and a lot of local state.  If this is used in a context like a list view, it could make a user effecting level of impact. Perhaps make this something passed to the view via the constructor.